alignak/satellite.py

```diff
@@ -83,6 +83,7 @@
 from alignak.daemon import Daemon
 from alignak.stats import statsmgr
 from alignak.check import Check  # pylint: disable=W0611
+from alignak.objects.module import Module  # pylint: disable=W0611
```

Comment:
Obviously it is not enough ... I thought it was, 

It is not logical, I agree, but after some more tests, this import seems to be needed here :/